quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerConfigPropertiesTest.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.quarkus.jdt.core;
+
+import static com.redhat.quarkus.jdt.internal.core.QuarkusAssert.assertProperties;
+import static com.redhat.quarkus.jdt.internal.core.QuarkusAssert.assertPropertiesDuplicate;
+import static com.redhat.quarkus.jdt.internal.core.QuarkusAssert.p;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.redhat.quarkus.commons.QuarkusProjectInfo;
+import com.redhat.quarkus.commons.QuarkusPropertiesScope;
+
+/**
+ * Test collect Quarkus properties from @ConfigProperties
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class JDTQuarkusManagerConfigPropertiesTest extends BaseJDTQuarkusManagerTest {
+
+	private static final int EXPECTED_PROPERTIES_FROM_GREETING_INTERFACE = 3;
+	private static final int EXPECTED_PROPERTIES_FROM_PUBLIC_FIELDS = 2;
+	private static final int EXPECTED_PROPERTIES_FROM_GETTER = 3;
+	private static final int EXPECTED_PROPERTIES_FROM_NO_PREFIX = 2;
+	private static final int EXPECTED_PROPERTIES_FROM_STACK_OVERFLOW = 2;
+
+	private final int EXPECTED_PROPERTIES = EXPECTED_PROPERTIES_FROM_GREETING_INTERFACE
+			+ EXPECTED_PROPERTIES_FROM_PUBLIC_FIELDS + EXPECTED_PROPERTIES_FROM_GETTER
+			+ EXPECTED_PROPERTIES_FROM_NO_PREFIX + EXPECTED_PROPERTIES_FROM_STACK_OVERFLOW;
+
+	@Test
+	public void configProperties() throws Exception {
+
+		QuarkusProjectInfo infoFromJavaSources = getQuarkusProjectInfoFromMavenProject("config-properties",
+				QuarkusPropertiesScope.sources);

```

Comment:
any known examples of ConfigProperties to be read from binaries?

Reply:
No it's like ConfigProperty, not found Quarkus extension which uses ConfigProperties annotations.